kroxylicious-systemtests/src/test/resources/kafka-admin-client-template.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: %NAME%
+  labels:
+    app: %NAME%
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: %NAME%
+  template:
+    metadata:
+      labels:
+        app: %NAME%
+      name: %NAME%
+      namespace: %NAMESPACE%
```

Comment:
why specify the namespace?

Reply:
to use the same namespace than kroxylicious and be able to remove it together with the rest of the pods that need to be removed after each test case

Reply:
That's not be point.  The code creates the deployment in the desired namespace.  Kubernetes will create the pod in the same namespace.  Adding it to the pod template within the deployment seems redundant.

Reply:
Got it. The file has been removed in the end